packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/mobile-component-library",
-  "version": "0.21.1-alpha.21",
+  "version": "0.21.1-alpha.22",
```

Comment:
Published an alpha version of the component library by mistake. Shouldn't have any affect but this is the latest alpha so leaving it.